app/locales/en/translations.js

```diff
@@ -217,14 +217,14 @@ export default {
         },
         'moderation-list-row': {
             submission: {
-                submitted_on: 'submitted on',
-                was_accepted_on: 'was accepted on',
-                was_rejected_on: 'was rejected on',
-                submitted: 'submitted',
-                was_accepted: 'was accepted',
-                was_rejected: 'was rejected',
-                by: 'by',
-                submission_by: 'submission by',
+                submitted_on: 'submitted on {{timeDate}} by',
+                submitted: 'submitted {{timeDate}} by',
+                accepted_on: 'accepted on {{timeDate}} by {{moderatorName}}',
+                accepted: 'accepted {{timeDate}} by {{moderatorName}}',
+                accepted_automatically_on: 'accepted automatically on {{timeDate}}',
+                accepted_automatically: 'accepted automatically {{timeDate}}',
+                rejected_on: 'rejected on {{timeDate}} by {{moderatorName}}',
```

Comment:
The keys in this file are inconsistent (and mostly snake_case), but I think we should start using camelCase for anything we touch.